.github/workflows/lint.yml

```diff
@@ -40,11 +40,17 @@ jobs:
         go mod download github.com/cockroachdb/crlfmt
         go install github.com/cockroachdb/crlfmt
 
-    - name: lint go files
+    - name: format go files
       run: |
         find . -name *.go -type f | xargs -n1 crlfmt -wrap=80 -ignore '_generated.deepcopy.go$$' -w
         git diff --exit-code
 
+    - name: lint rpk
+      uses: golangci/golangci-lint-action@v3
+      with:
+        version: v1.40.1
```

Comment:
Why v1.40.1? I think the current version is 1.45.2

Reply:
No real reason, just wanted to keep consistency with vtools. Thanks! Updgraded 👍